.github/scripts/build-windows.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+choco install -y wasmedge
```

Comment:
Is there a way to specify the version?
Could you create an issue in [wasmedge-rust-sdk](https://github.com/WasmEdge/wasmedge-rust-sdk/issues) to support standalone Windows?

Reply:
https://github.com/WasmEdge/wasmedge-rust-sdk/issues/54

Reply:
pinned the version